crates/cli/tests/update_tests.rs

```diff
@@ -0,0 +1,42 @@
+use std::fs;
+
+use ndc_postgres_cli::*;
+use ndc_postgres_configuration as configuration;
+use ndc_postgres_configuration::RawConfiguration;
+
+const CONNECTION_STRING: &str = "postgresql://postgres:password@localhost:64002";
+
+#[tokio::test]
+async fn test_update_configuration() -> anyhow::Result<()> {
```

Comment:
When does this run? As part of `just dev` and `just test`?

Reply:
Yeah, it's a normal test case; connects to the database just like the query tests do.